superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -125,3 +127,62 @@ def _pre_action(self) -> None:
     def _migrate_temporal_filter(self, rv_data: dict[str, Any]) -> None:
         super()._migrate_temporal_filter(rv_data)
         rv_data["adhoc_filters_b"] = rv_data.get("adhoc_filters") or []
+
+
+class TimeseriesChart(MigrateViz):
+    has_x_axis_control = True
+
+    def _pre_action(self) -> None:
+        self.data["contributionMode"] = "row" if self.data.get("contribution") else None
+
+        show_brush = self.data.get("show_brush")
+        self.data["zoomable"] = False if show_brush == "no" else True
+
+        self.data["markerEnabled"] = self.data.get("show_markers")
```

Comment:
What if `show_markers` is undefined? Can `markerEnabled` be a tri-state?

Reply:
Changed it to use `False` by default.